### DISCUSSION TITLE
I got my student GitHub benefits, but still not able to use it

### DISCUSSION THREAD

**Discussion by PhysicsAbdelhamid** [community/community#112077] [GitHub Education] [answered]
### Student GitHub benefits

Dear GitHub community, 

As said in the title, my request for a student GitHub benefit was granted few days ago, I am connected to my GitHub account in VS studio, I also downloaded the GitHub copilot extension in it. 

But still, when I click on "CHAT:GITHUB COPILOT", it still suggest me to "start the 30-day free  trial", and there is no auto-completion suggestions at all. Have I missed something please ? how much time on average should I wait? Or maybe I am not testing it in the correct way at all ?

Ps. I don't know if this can cause a difference, but I am trying to use it when working in a remote ssh server.

Thank you for any kind of help.

**Comment by moozunch** [comment]
![image](https://github.com/community/community/assets/112236945/4580c5a8-e55e-47b1-a9f7-b696807b9fe8)
try checking your copilot subscription, does it give you the copilot subscription, if it does it will appear like attachment image and probably try to follow the instruction for your IDE, I follow the instruction and it works just fine for me as I also using Github for Student and it gives me pro plan for my Github account ^__^

**Reply by PhysicsAbdelhamid** [reply]
Hi Annisa, 

Thanks for prompt reply. It seems that this is missing in my profile, do you remember if you had to wait a bit from the time your account was granted student package and the time it was effectively working ? 


![Screenshot from 2024-03-11 08-24-06](https://github.com/community/community/assets/114695726/3176b344-8a08-45b5-85b2-756de645fd00)

**Comment by davecheney** [comment, marked as answer]
@PhysicsAbdelhamid there is no students-2024 coupon attached to this github handle. Please contact GitHub Education, we cannot assist you via this forum.

**Reply by PhysicsAbdelhamid** [reply]
Hi @davecheney, thanks for letting me know, I will close the question and proceed with them. Have a nice day/evening.
